### DISCUSSION TITLE
SV06Plus ASA and TPU profile

### DISCUSSION THREAD

**Discussion by madhat81** [OrcaSlicer/OrcaSlicer#1608] [Q&A]
Hi. I wonder why there are no profiles for ASA and TPU for the Sovol SV06Plus. I like the Orca Slicer, but without the ability to print ASA and TPU it's useless for me. Thanks for your help

**Comment by hliebscher** [comment]
You can use any filament with any Printer model... edit any filament with temp and other parameter Save as TPU thats it!

**Reply by madhat81** [reply]
But there are other differences between the filaments. Like retraction, density etc.
